apps/native/app/src/utils/permissions.ts

```diff
@@ -0,0 +1,30 @@
+import messaging from '@react-native-firebase/messaging'
+import { PermissionsAndroid } from 'react-native'
+import { authStore } from '../stores/auth-store'
+import { androidIsVersion33OrAbove } from './versions-check'
+
+export const requestAndroidPostNotificationsPermission = async () => {
+  const granted = await PermissionsAndroid.request(
+    PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS,
+  )
+
+  return granted === PermissionsAndroid.RESULTS.GRANTED
+}
+
+export const requestPostNotificationsPermission = async () => {
```

Comment:
~why PostNotifications? instead of requestNotificationsPermission

Reply:
Good point was referring to the actual permission which is POST_NOTIFICATIONS